libs/wingc/src/type_check.rs

```diff
@@ -5750,6 +5750,10 @@ impl<'a> TypeChecker<'a> {
 			} else {
 				var.clone()
 			}
+		} else if let LookupResult::NotFound(property) = lookup_res {
+			self
+				.spanned_error_with_var(&property, format!("Property \"{property}\" does not exist"))
+				.0
```

Comment:
This will work, but what I don't like about it is that it created `lookup_result_to_type_error` specifically to generate meaningful error's based on a `LookupResult` but here we added an exception and handled the `NotFound` variant in a different way than what's currently implemented in `lookup_result_to_type_error`.
I think a better approach would be to extend how `lookup_result_to_type_error` handles `NotFound` conditions so it knows if the looked up symbol is a `ClassLike` property or not and produce the correct diagnostic.
So what I'd try is to add another argument to `lookup_result_to_type_error` which is the environment (`env`) in which the lookup happened. If that env turns out to be a class env we can say this is a property lookup error, and if it's not a class env then we can say this is an "unknown symbol". Makes sense?
Do you want to try this?

Reply:
Makes sense! I was considering modifying `lookup_result_to_type_error` but decided against it since I figured it'd be a larger change. I agree that doing so would be a cleaner solution though. I'll give it a try.

Reply:
I tried this approach but wasn't able to make any progress getting the information out of `env`. At this point, it'd probably take less time to write the code than to guide me to a solution, so I'll close this PR and stick to the areas I understand better.

Reply:
Ok. Hope it wasn't too frustrating. I'll take a look. Thanks a lot for giving it a try!